From bf17e040101a51cda34ed246768a175afabd9a14 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Marco=20Trevisan=20=28Trevi=C3=B1o=29?= <mail@3v1n0.net>
Date: Tue, 15 Oct 2019 14:38:14 +0000
Subject: [PATCH 24/50] clutter/stage: Actually set key focus to an actor on
 key focus

As per commit c2d03bf73 we're using a private method to set the actors key
focus and emit key-focus-in signal, but we're using inverted logic here.

So flip the parameter to match the expected result.

https://gitlab.gnome.org/GNOME/mutter/merge_requests/860


(cherry picked from commit 553211dd81a849f33f0aca447c8683f031fff868)
---
 clutter/clutter/clutter-stage.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/clutter/clutter/clutter-stage.c b/clutter/clutter/clutter-stage.c
index 3a8a019d2..c4b88b32e 100644
--- a/clutter/clutter/clutter-stage.c
+++ b/clutter/clutter/clutter-stage.c
@@ -3064,7 +3064,7 @@ clutter_stage_set_key_focus (ClutterStage *stage,
   if (actor != NULL)
     {
       priv->key_focused_actor = actor;
-      _clutter_actor_set_has_key_focus (actor, FALSE);
+      _clutter_actor_set_has_key_focus (actor, TRUE);
     }
   else
     _clutter_actor_set_has_key_focus (CLUTTER_ACTOR (stage), TRUE);
-- 
2.23.0

